utils/src/main/kotlin/Utils.kt

```diff
@@ -198,15 +198,12 @@ fun getPathFromEnvironment(executable: String): File? {
  */
 fun fixupUserHomeProperty() {
     val userHome = System.getProperty("user.home")
-    val checkedUserHome = sequenceOf(
-        userHome,
-        Os.env["HOME"],
-        Os.env["USERPROFILE"]
-    ).find {
-        !it.isNullOrBlank() && it != "?"
+    if (userHome == null || userHome == "?") {
+        sequenceOf(
```

Comment:
I have to admit it's not really beneficial to use a sequence here, so we could as well use `listOfNotNull` here instead of `sequenceOf()` and `filterNotNull`.

Reply:
Done.